fuse_core/include/fuse_core/throttled_callback.h

```diff
@@ -161,9 +169,16 @@ class ThrottledCallback
     // (a) This is the first call, i.e. the last called time is still invalid because it has not been set yet
     // (b) The throttle period is zero, so we should always keep the callbacks
     // (c) The elpased time between now and the last called time is greater than the throttle period
-    #warn "using a valid time value as a flag"
-    const rclcpp::Time now = use_wall_time_ ? rclcpp::Clock::Clock(RCL_SYSTEM_TIME).now() : node->now();
-    if ((last_called_time_.nanoseconds() == 0) || (throttle_period_.nanoseconds() == 0) || now - last_called_time_ > throttle_period_)
+    rclcpp::Time now;
+    if (use_wall_time_) {
+      now = rclcpp::Clock(RCL_SYSTEM_TIME).now();
```

Comment:
This seems to be constructing a `Clock` every callback. I recommend storing the clock on the `ThrottledCallback` class instead since it seems to be needed for the life of the instance.

Reply:
https://github.com/locusrobotics/fuse/pull/286/commits/d9493d49276c9dcab014a65aed5ffbf15aa7d714